README.md

```diff
@@ -13,6 +13,42 @@
     - [Rails](#rails)
     - [Servers](#servers)
     - [Additional Resources](#additional-resources)
+  - [Product Management](#product-management)
+    - [General Principles](#general-principles)
+    - [Ticket Tracking Workflow (JIRA or similar)](#ticket-tracking-workflow)
+    - [GitHub Workflow](#github-workflow)
+
+## Product Management
+
+### General Principles
+
+* **Over-communicate**: It should be impossible for team leads or clients to be unsure about what you are working on, challenges you are facing, questions you have, assumptions you've made, etc. We employ Loud Productivity<sup>TM</sup> to ensure clients and team members can "hear" how projects are going without having to look closely or investigate to understand what is happening.
+* **Disambiguate Responsibility**: It should be obvious at every step in the life cycle of a JIRA ticket or GitHub PR who is immediately responsible for its completion. See the workflows below regarding ticket assignments and related expectations.
+* **Anticipate Challenges**: It is incumbent upon the developer in charge of a ticket to *understand the business requirements and implementation plan thoroughly ahead of the implementation* and to identify/raise concerns or questions before any code is written. We are frequently 12 time zones away from clients, so frequent back-and-forths about minor issues/concerns throughout the implementation of any given feature require 24-hour cycles and is _devastating_ to timelines and productivity. Ask blocking questions as early as possible to minimize their quantity and their impact on the timeline of the feature. If possible, propose solutions or seek clarification internally before engaging clients directly for input.
+
+### Ticket Tracking Workflow
+
+* Use JIRA (or a similar PM software) to track discrete business requirements and the process from discovery through completion.
+* Use columns that capture the following intents (at minimum):
+  1. TODO: Tickets that are unassigned but ready for immediate development. These should be estimated (via story points or comparable measure) prior to or soon after being added to the TODO column. Estimate and/or self-assign an unassigned TODO ticket as soon as you are ready to begin development on that ticket.
+  2. IN PROGRESS: This column indicates that **development is active**. Make sure that whatever you are currently working on is in this column and assigned to you. There should never be multiple in-progress tickets assigned to you unless they are in fact both in active development (and potentially should have been one ticket). If something needs clarification or review it _should not be in this column_.
```

Comment:
@hiattp worth clarifying: if a ticket gets blocked after it's started (needs additional info, decision needs to be made by the client, waiting for access credentials, etc) where should it be?

Reply:
Yep good call. I think it should be in the TODO (or whichever prior column) and assigned to whomever is blocking, with a comment on the ticket articulating what is needed and who identified the block. It might also be appropriate to put it in the IN REVIEW column if the block depends on a review, to indicate that the ticket is mostly done (or at least work as started) but input is needed either on the implementation or UX before continuing. Thoughts?

Reply:
I agree with @hiattp on this one, it's what I usually do/suggest.